src/js/components/LongDashPlaceholder.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
```

Comment:
I understand the need for a means to not hardcode the same unicode everywhere but it feels strange to create component to only render one character. Please note that every component comes with an overhead.

Reply:
Understood, those are good options too although I rather have a composable UI.

I'm changing to one of your suggestions though so we can merge the PR.

Reply:
I'm aspiring a declarative and compose-able UI as well, but honestly don't think that `<LongDashPlaceholder />` is any better than simply returning a long dash, which is why I'd go for a constant or a general purpose placeholder util or component.

You are right that some of our the components should, based on the presented logic, indeed be constants and helpers as well. I actually think that we should refactor those components as wouldn't say that writing these as react components is in any way more declarative or compose-able. There is a reason why the community is pushing functional components which are basically nothing else then simple helper functions.

Reply:
addressed using util. thanks.